File: Data/Data.Models/Entities/Review.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Data.Models.Entities
{
    public class Review
    {
        [JsonPropertyName("id")]
        [Key]
        public int Id { get; set; }

        [JsonPropertyName("reviewerEmail")]
        public String ReviewerEmail { get; set; }

        [JsonPropertyName("revieweeEmail")]
        public String RevieweeEmail { get; set; }

        [JsonPropertyName("content")]
        public String Content { get; set; }
    }
}

File: Logic/src/models/Review.java
package models;

public class Review {
    private int id;
    private String reviewerEmail;
    private String revieweeEmail;
    private String content;

    public Review(int id, String reviewerEmail, String revieweeEmail, String content)

    {
        this.id = id;
        this.revieweeEmail = revieweeEmail;
        this.reviewerEmail = reviewerEmail;
        this.content = content;


    }

    public int getId() {
        return id;
    }

    public String getReviewerEmail() {
        return reviewerEmail;
    }

    public String getRevieweeEmail() {
        return revieweeEmail;
    }

    public String getContent() {
        return content;
    }
}
